File: 백준/Bronze/2566. 최댓값/최댓값.java
import java.io.*;
import java.util.*;

class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int[][] arr = new int[9][9];
        int max = arr[0][0];
        int x=0, y=0;

        for(int i=0; i<arr.length; i++) {
            StringTokenizer st = new StringTokenizer(br.readLine());
            for(int j=0; j<arr.length; j++) {
                arr[i][j] = Integer.parseInt(st.nextToken());
                if(arr[i][j]>max) {
                    max=arr[i][j];
                    x=i;
                    y=j;
                }

            }
        }
        System.out.println(max);
        System.out.println((x+1) + " " + (y+1));
    }
}

File: 백준/Bronze/2562. 최댓값/최댓값.kt
import java.io.*
import java.util.*

fun main() {
    val br = BufferedReader(InputStreamReader(System.`in`))

    val intArray = IntArray(9)

    for (i in 0 until 9) {
        intArray[i] = Integer.parseInt(br.readLine())
    }
    println(findMaxNumbers(intArray))


}

fun findMaxNumbers(arr: IntArray): String {
    var max = arr.max()
    var maxIndex = arr.indexOf(max) + 1

    return "$max\n$maxIndex"
}

